### PR TITLE
Add python bindings test to travis

### PR DESCRIPTION
**Pull request by pranith**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ before_install:
 script:
         - ./make.sh
         - make check
+        - cp libcapstone.so bindings/python/
```
Comment (aquynh) [inline .travis.yml]: this "libcapstone.so" limits our test to Linux only. OSX has "libcapstone.dylib", can we support that?

Comment (pranith) [inline .travis.yml]: let me try to install the library as before...

Comment (aquynh) [inline .travis.yml]: Great, after that we can add OSX test there.

Comment (aquynh) [inline .travis.yml]: a simple trick is to replace that line with:

    - cp libcapstone.* bindings/python/

then we can support both Linux & OSX.

**Comment by aquynh** [comment]
merged with this fix for OSX support https://github.com/aquynh/capstone/commit/0989f651ccddcb6847aa21f9801aa99a42954a47, thanks!

**Pull request merged**